### PR TITLE
John/slater porous bleed

### PR DESCRIPTION
**Pull request by johnmoore4**

### CODE REVIEW

File: flow360/component/simulation/models/surface_models.py [hunk 1]
```diff
@@ -131,6 +131,27 @@ class Pressure(SingleAttributeModel):
     value: PressureType.Positive = pd.Field(description="The static pressure value.")
 
 
+class SlaterPorousBleed(Flow360BaseModel):
+    """
+    :class:`SlaterBleedPressure` class to specify the static pressure of the Slater bleed model
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:136]: This is not consistent with the class name

File: flow360/component/simulation/models/surface_models.py [hunk 2]
```diff
@@ -131,6 +131,27 @@ class Pressure(SingleAttributeModel):
...
+    `Outflow` boundary condition via :py:attr:`Outflow.spec`.
+
+    Example
+    -------
+
+    >>> fl.SlaterPorousBleed(value = 1.01e6 * fl.u.Pa)
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:142]: Please update example and describe its physical meaning.

File: flow360/component/simulation/models/surface_models.py [hunk 3]
```diff
@@ -131,6 +131,27 @@ class Pressure(SingleAttributeModel):
...
+
+    ====
+    """
+
+    type_name: Literal["SlaterPorousBleed"] = pd.Field("SlaterPorousBleed", frozen=True)
+    # pylint: disable=no-member
+    static_pressure: PressureType.Positive = pd.Field(description="The static pressure value.")
+    porosity: float = pd.Field(
+        gt=0, le=1, description="The porosity of the bleed region. Range [0,1]."
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:151]: No need for hand-written range. Documentation will handle it.
```suggestion
        gt=0, le=1, description="The porosity of the bleed region."
```

File: flow360/component/simulation/models/surface_models.py (hunk 1 above)
Comment (benflexcompute) [reply flow360/component/simulation/models/surface_models.py:136]: Also could you be elaborate a bit on what this model is exactly?

File: flow360/component/simulation/models/surface_models.py (hunk 3 above)
Comment (benflexcompute) [reply flow360/component/simulation/models/surface_models.py:151]: Do we need default values for porosity or static_pressure?

File: flow360/component/simulation/translator/solver_translator.py [hunk 4]
```diff
@@ -791,6 +792,12 @@ def boundary_spec_translator(model: SurfaceModelTypes, op_acousitc_to_static_pre
             boundary["staticPressureRatio"] = (
                 model_dict["spec"]["value"] * op_acousitc_to_static_pressure_ratio
             )
+        elif isinstance(model.spec, SlaterPorousBleed):
```
Comment (benflexcompute) [inline flow360/component/simulation/translator/solver_translator.py:795]: Can you add a unit test in translator to test that this kind of boundary is properly translated?

Comment (benflexcompute) [reply flow360/component/simulation/translator/solver_translator.py:795]: https://github.com/flexcompute/Flow360/blob/develop/tests/simulation/translator/test_solver_translator.py
